TinkeringGraphics

```diff
@@ -0,0 +1,111 @@
+#To start the prog type:   main()
+
+#Turns the image black and white
+def blackAndWhite(image):
+   for pixel in image.getPixels():
+      red = pixel.getRed()
+      green = pixel.getGreen()
+      blue = pixel.getBlue()
+      average = (red + green + blue) / 3
+      if average < 128:
+         red = green = blue = 0
+      else:
+         red = green = blue = 255
+      pixel.setRed(red)
+      pixel.setGreen(green)
+      pixel.setBlue(blue)
+#add a grid to the image
```

Comment:
Include sufficient whitespace between functions. Follow the [PEP-8 guidelines](https://www.python.org/dev/peps/pep-0008/#blank-lines).